### ISSUE TITLE
Inheritance options don't work when randomizing Pokemon

### ISSUE THREAD

**Issue by drkz4ck** [kwsch/pkNX#175]
Tried to mix and match these settings in many ways to no avail.
Either it seems that the inherit options don't work atm or I'm missing a really important step here

**Settings**
![image](https://user-images.githubusercontent.com/76820046/143273227-60fa24ba-b698-4393-a43f-9b52f0b47b34.png)

**Attempt1**
![image](https://user-images.githubusercontent.com/76820046/143273387-eef9b0db-268d-4d12-b574-7dc21d0b3db4.png)
![image](https://user-images.githubusercontent.com/76820046/143273428-51dc3947-b388-4dbf-a3f7-ba6d7b299e79.png)
___________________
**Attempt2**
![image](https://user-images.githubusercontent.com/76820046/143273548-26eabd6f-411b-4caf-bbae-516867745892.png)
![image](https://user-images.githubusercontent.com/76820046/143273587-ce81c10e-6f9c-4db2-badb-d9687bc63332.png)

**Comment by kwsch** [comment]
Did you randomize evolutions beforehand?

**Comment by drkz4ck** [comment]
No, even if it's the very first randomization performed it won't work as intended.
I've provided a video performing the task from scratch

https://user-images.githubusercontent.com/76820046/143306863-508fd554-de28-46bc-8f85-370722557887.mp4

**Comment by Lusamine** [comment]
Closing because this was fixed in the PR above.
